integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -276,6 +286,17 @@ void testCreateMiiDomain() {
       verifyCredentials(adminServerPodName, domainNamespace,
             ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT, true);
     }
+
+    // Test that `kubectl port-foward` is able to forward a local port to default channel port (7001 in this test)
+    // and default secure channel port (7002 in this test)
+    // Verify that the WLS admin console can be accessed using http://localhost:localPort/console/login/LoginForm.jsp
```

Comment:
Verify that the WLS admin console can be accessed using --> Verify that the WLS admin console can not be accessed using